argonaut/src/main/scala/org/http4s/argonaut/ArgonautInstances.scala

```diff
@@ -18,11 +18,11 @@ trait ArgonautInstances {
     }
 
   implicit val jsonEncoder: EntityEncoder[Json] =
-    EntityEncoder.stringEncoder(Charset.`UTF-8`).contramap[Json] { json =>
```

Comment:
I feel like I'm nagging, but can we keep the explicit charset definition? The default is UTF-8, but keeping it explicit will future proof a bit in case the sky falls and UTF-8 falls out of fashion and we change the default definition and makes it easier to _know_ the string will be UTF-8 encoded without having to look in the package object.